Session/Home/HomeViewModel.swift

```diff
@@ -52,18 +53,48 @@ public class HomeViewModel: NavigatableStateHolder {
         let retryCount = dependencies[defaults: .standard, key: .rateAppRetryAttemptCount]
         
         var promptState: AppReviewPromptState?
+        
+        // A buffer of 24 hours
+        let buffer: TimeInterval = 24 * 60 * 60
 
-        if retryCount == 0, let retryDate = dependencies[defaults: .standard, key: .rateAppRetryDate], dependencies.dateNow >= retryDate {
+        if retryCount == 0, let retryDate = dependencies[defaults: .standard, key: .rateAppRetryDate], dependencies.dateNow.timeIntervalSince(retryDate) >= -buffer {
```

Comment:
This logic is now getting fairly complex for `HomeViewModel.init` - can you please add a static function: `AppReviewPromptModel.loadInitialState` which returns a tuple of `AppReviewPromptState?` and put this logic in there?